corehq/apps/builds/models.py

```diff
@@ -239,6 +258,19 @@ def get_menu(self, application_version=None):
         else:
             return self.menu
 
+    @classmethod
+    @quickcache([], timeout=5 * 60)
+    #This seems to be not working.
+    def j2me_enabled_configs(cls):
+        return [build for build in cls.fetch().menu if build.j2me_enabled]
+
+    @classmethod
+    def j2me_enabled_configs_labels(cls):
```

Comment:
Nitpick: I'd also call this `j2me_enabled_config_labels` (not config**s** labels)